Gems/Atom/Tools/AtomToolsFramework/Code/Source/Document/AtomToolsDocumentMainWindow.cpp

```diff
@@ -346,6 +346,10 @@ namespace AtomToolsFramework
                 // extensions displayed in the dialog
                 const QString name = tr("Open %1 Document...").arg(documentType.m_documentTypeName.c_str());
                 CreateActionAtPosition(parentMenu, insertPostion, name, [documentType, toolId = m_toolId]() {
+                    // Visual Studio 2022 flags toolId as unused even though it is passed to the QueueFunction lambda below.
+                    // Use AZ_UNUSED to prevent the compiler error.
+                    AZ_UNUSED(toolId);
```

Comment:
Yeah I have seen VS2022 incorrectly flag lambda captured variables as unused before in the NameTests